ScoutSuite/output/data/inc-scoutsuite/scoutsuite.js

```diff
@@ -374,6 +382,35 @@ function toggleDetails(keyword, item) {
     $(id).toggle();
 };
 
+/**
+ * Toggles between light and dark themes
+ */
+function toggle_theme() {
+    if (document.getElementById("dark_theme").checked) {
+        this.set_theme(DARK_THEME)
+    }
+    else {
+        this.set_theme(LIGHT_THEME)
+    }
+};
+
+/**
+ * Sets the css file location received as the theme
+ * @param file
+ */
+function set_theme(file)
+{
+    var oldlink = document.getElementsByTagName("link").item(THEME_INDEX);
+    oldlink.href = file;
+}
+
+/**
+ * Save the current theme on web storage
+ */
+window.onunload = function() {
+    localStorage.setItem("dark_theme", document.getElementById("dark_theme").checked);
```

Comment:
I wonder if this is compliant with the Cookie Law.

> The Cookie Law is a piece of privacy legislation that requires websites to get consent from visitors to store or retrieve any information on a computer, smartphone or tablet.

Reply:
I don't think that's an issue as it's not a website, just some local html being rendered in the browser.